### ISSUE TITLE
 Increased circulating very long-chain fatty acid concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6719]
**Preferred term label:**
 Increased circulating very long-chain fatty acid concentration

**Synonyms**
 Increased plasma levels of very long-chain fatty acid

**Definition (free text, please give PubMed ID)**
Increased concentation of very long-chain fatty acids in the blood circulation. Very long-chain fatty acids are fatty acids (FAs) with a chain-length of 22 or more carbons.

**Parent term (use hpo.jax.org/app)**
Abnormal circulating fatty-acid concentration HP:0004359

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:261515 
PMID: 9345094  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
